pkgs/by-name/op/opencloud-desktop/package.nix

```
@@ -3,13 +3,31 @@
   cmake,
   kdePackages,
   fetchFromGitHub,
+  fetchFromGitLab,
   libre-graph-api-cpp-qt-client,
   kdsingleapplication,
   nix-update-script,
   qt6,
   versionCheckHook,
   stdenv,
 }:
+let
+  # See issue https://github.com/NixOS/nixpkgs/issues/455981
+  # It was spotted upstream that ecm 6.19.0 caused this issue
+  # Therefore we set ecm to 6.18.0 which makes opencloud open and work as expected
```

Alternative wording suggestion:

```suggestion
  # Therefore we pin ecm 6.18.0 which makes opencloud function correctly
```